cynic-querygen/src/schema/fields.rs

```diff
@@ -193,7 +193,7 @@ fn input_type_spec_imp(
             needs_owned,
             is_subobject_with_lifetime,
         )
-        .map(|type_spec| format!("Option<{type_spec}>",));
+        .map(|type_spec| format!("cynic::MaybeUndefined<{type_spec}>",));
```

Comment:
I don't want to force every user of the generator into using MaybeUndefined.  I'd rather this was controlled by a parameter somewhere - either a parameter passed in rust or maybe a directive in the query that we're generating from?